packages/shared/styles/components/organisms/SfCollectedProduct.scss

```
@@ -15,78 +15,75 @@
     --collected-product-background: var(--c-white);
   }
   &__remove {
-    --button-padding: 0;
-    --icon-size: 0.875rem;
-    --icon-color: var(--c-gray);
+    --icon-size: 8px;
+    --icon-color: var(--c-light-variant);
     position: absolute;
     opacity: var(--collected-product-remove-opacity);
-    top: var(--collected-product-remove-top, var(--spacer));
-    right: var(--collected-product-remove-right, var(--spacer));
+    top: var(--collected-product-remove-top);
+    right: var(--collected-product-remove-right, 0);
     transform: var(--collected-product-remove-transform);
-    background-color: var(--collected-product-remove-background, transparent);
+    background: var(--collected-product-remove-background, transparent);
     box-shadow: var(--collected-product-remove-box-shadow);
     transition: var(--collected-product-remove-transition);
-    &:hover,
-    &:active {
-      --collected-product-remove-box-shadow: none;
-      --collected-product-remove-background: transparent;
+    &:hover {
+      --collected-product-remove-background: var(--c-primary-variant);
     }
   }
+  &__more-actions {
+    --icon-width: 4px;
+    --icon-height: 15px;
+    position: absolute;
+    top: var(--collected-product-remove-top);
+    right: var(--collected-product-remove-right, 0);
+  }
   &__aside {
     position: relative;
-    flex: 0 0 5.3125rem;
+    flex: 0 0 8.75rem;
   }
   &__image {
     background: var(--collected-product-image-background, var(--c-light));
   }
-  & &__quantity-selector {
-    --input-background: var(--c-white);
+  &__quantity-selector {
+    --quantity-selector-background: var(--c-light-variant);
+    --quantity-selector-height: 2.5rem;
+    --quantity-selector-width: 7rem;
     position: absolute;
-    right: var(--spacer);
-    bottom: var(--spacer);
+    right: var(--spacer-sm);
+    bottom: var(--spacer-sm);
     z-index: 1;
   }
   &__main {
     flex: 1;
-    margin: var(--collected-product-main-margin, 0 0 0 var(--spacer-big));
+    margin: var(--collected-product-main-margin, 0 0 0 var(--spacer-base));
     display: flex;
     flex-direction: column;
   }
   &__title {
-    margin-bottom: 5px;
-    font: var(
+    margin-bottom: var(--spacer-base);
+    @include font(
       --collected-product-title-font,
-      font(
-        --collected-product-title,
-        var(--body-font-weight-primary),
-        var(--font-size-regular),
-        1.6,
-        var(--font-family-secondary)
-      )
+      var(--font-normal),
+      var(--font-2xs),
+      1.6,
+      var(--font-family-secondary)
     );
   }
   &__details {
     margin: auto 0 0 0;
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-end;
   }
   @include for-desktop {
+    --collected-product-max-width: 350px;
     --collected-product-padding: 10px;
     --collected-product-remove-top: 10px;
     --collected-product-remove-right: 0;
     --collected-product-remove-transform: translateX(50%);
```

wdyt? we use it very often, maybe we can stick to one form? :)
```suggestion
    --collected-product-remove-transform: translate3d(50%, 0, 0);
```